app/services/CategorisationService.scala

```diff
@@ -56,7 +59,9 @@ class CategorisationService @Inject() (
                                       getGoodsRecordResponse.countryOfOrigin,
                                       LocalDate.now() //TODO where does DateOfTrade come from??
                                     )
-          categorisationInfo     <- Future.fromTry(Try(CategorisationInfo.build(goodsNomenclature).get))
+          originalCommodityCode   = originalCommodityCodeOpt.getOrElse(getGoodsRecordResponse.commodityCode)
+          categorisationInfo     <-
+            Future.fromTry(Try(CategorisationInfo.build(goodsNomenclature, Some(originalCommodityCode)).get))
```

Comment:
.get

Reply:
Done :) Committing soon :)

Reply:
Changed it, but I preferred it with the .get as it's inside a future from try anyway. Committing soon :)